controllers/articles.controller.js

```diff
@@ -0,0 +1,12 @@
+const { selectArticleByID } = require("../models");
+
+exports.getArticleByID = (req, res, next) => {
+  const { article_id } = req.params;
+  selectArticleByID(article_id)
+    .then((article) => {
+      res.status(200).send({ article: article });
```

Comment:
can be just .send({article}). This will make an object with a key of article, and make it's value the same value assigned to the 'article' variable  returned from your then block.